recipe/meta.yaml

```diff
@@ -32,4 +32,4 @@ about:
 
 extra:
   recipe-maintainers:
-    -janschulz
\ No newline at end of file
+    - janschulz
```

Comment:
Will [these](https://github.com/conda-forge/conda-forge.github.io/pull/43/files#diff-2f496464fa7193d7f4a20f02b3b17f1dR108) changes have any impact in the GitHub API?